SECURITY.md

```diff
@@ -6,8 +6,9 @@ The versions of the project that are currently supported with security updates.
 
 | Material UI version | Release    | Supported                            |
 | ------------------: | :--------- | :----------------------------------- |
-|              ^6.0.0 | 2024-08-26 | :white_check_mark: Stable major      |
-|              ^5.0.0 | 2021-09-16 | :white_check_mark: Long-term support |
+|              ^7.0.0 | 2025-03-26 | :white_check_mark: Stable major      |
+|              ^6.0.0 | 2024-08-26 | :white_check_mark: Long-term support |
+|              ^5.0.0 | 2021-09-16 | :x:                                  |
```

Comment:
An LTS of 8 months, I don't think that we can do this. 

```suggestion
|              ^5.0.0 | 2021-09-16 | :white_check_mark: Long-term support |
```

Similar comment as in https://github.com/mui/material-ui/pull/45670#discussion_r2030239321. 

Up until now, the strategy was designed for: one major per year https://mui.com/versions/#release-frequency, previous in LTS, so LTS of 1 year. But https://mui.com/material-ui/getting-started/support/#long-term-support-lts says nothing about it; others do, e.g. https://nodejs.org/en/about/previous-releases#nodejs-releases

The last discussion on this was long ago https://www.notion.so/mui-org/Does-it-make-sense-to-have-an-LTS-version-eb623e33e9c340f2aa0134f503a6000e.

Reply:
I agree with the suggestion. LTS should be at least 2 versions back.

Reply:
If we support for 2 years, would that mean we support two long-term versions alongside the current major?

Reply:
@joserodolfofreitas if we release once a year, yes.

1. should it be 1 years or 2 years, 2 feels right considering the effort it takes to deliver it, and the migration speed the community has. I would suggest we make the change. We discussed this a bit with @mbrookes in the past.
2. In any cases, here, we don't even need to decide to change LTS from 1 years to 2 years. 1 years is already enough to keep Material UI v5 security bug fixes.

Reply:
Fine by me, is this correct?:

- v7: Current stable
- v6: LTS
- v5: LTS